app/domain/authentication/authn_jwt/vendor_configurations/configuration_jwt_generic_vendor.rb

```diff
@@ -21,7 +21,8 @@ def initialize(
           create_constraints: Authentication::AuthnJwt::RestrictionValidation::CreateConstrains.new,
           fetch_claim_aliases: Authentication::AuthnJwt::RestrictionValidation::FetchClaimAliases.new,
           validate_and_decode_token: Authentication::AuthnJwt::ValidateAndDecode::ValidateAndDecodeToken.new,
-          restrictions_from_annotations: Authentication::ResourceRestrictions::GetServiceSpecificRestrictionFromAnnotation.new
+          restrictions_from_annotations: Authentication::ResourceRestrictions::GetServiceSpecificRestrictionFromAnnotation.new,
+          restriction_validator: Authentication::AuthnJwt::RestrictionValidation::ValidateRestrictionName.new
```

Comment:
i already suggested to remove this but in general the name should be `validate_restriction_name`

Reply:
Renamed